### PR TITLE
ChecksFinder: Tweak link in ChecksFinder

### PR DESCRIPTION
**Pull request by TheLynk**
Please format your title with what portion of the project this pull request is
targeting and what it's changing.

change of link for the checksfinder Setup page for more continuity compared to the other setup page
https://archipelago.gg/tutorial/ChecksFinder/checksfinder/en --> https://archipelago.gg/tutorial/ChecksFinder/setup/en
## What is this fixing or adding?

change of link for the checksfinder Setup page for more continuity compared to the other setup page
https://archipelago.gg/tutorial/ChecksFinder/checksfinder/en --> https://archipelago.gg/tutorial/ChecksFinder/setup/en

### CODE REVIEW

File: worlds/checksfinder/__init__.py [hunk 1]
```diff
@@ -15,7 +15,7 @@ class ChecksFinderWeb(WebWorld):
         "single-player, multiworld, and related software.",
         "English",
         "checksfinder_en.md",
```
Comment (remyjette) [inline worlds/checksfinder/__init__.py:17]: You need to also change this to setup_en.md and rename the file.

> The file name you use aren't particularly important, but it should be descriptive of what
the tutorial is covering, and the name of the file must match the relative URL provided in the `Tutorial`. Currently,
the JS that determines this ignores the provided file name and will search for `game/document_lang.md`, where
`game/document/lang` is the provided URL.

https://github.com/ArchipelagoMW/Archipelago/blob/main/docs/world%20api.md#tutorials

Comment (TheLynk) [reply worlds/checksfinder/__init__.py:17]: okay go fix

**Review by remyjette — APPROVED** [review]
Confirmed it works now on my local webhost

**Comment by ThePhar** [comment]
@jonloveslegos

**Comment by jonloveslegos** [comment]
The change looks fine to me.

**Review by ThePhar — APPROVED** [review]

**Pull request merged**